packages/modules/vehicles/vwid/socutils.py

```diff
@@ -48,23 +32,23 @@ def write_token_file(self, path: str, token: str, config={}):
             self.tf.write(token)         # write Token file
             self.tf.close()
         except Exception as e:
-            log.exception('Token file write exception ' + str(e))
+            log.debug('Token file write exception ' + str(e))
```

Comment:
Warum änderst Du das wieder zurück? 
Die Logmeldungen in Exceptions sollten die log.exception-Methode aufrufen, damit automatisch und einheitlich die Exception geloggt wird. (Betrifft auch die anderen Stellen mit der gleichen Änderung)
```suggestion
            log.exception('Token file write exception ')
```